src/components/Dashboard/Dashboard.module.scss

```diff
@@ -119,15 +126,65 @@
 .btn {
   width: 45%;
   padding: 0.5rem;
-  color: $severely-burnt-toast;
-  background-color: $translucent-silk;
-  border: none;
+  color: $translucent-silk;
+  background-color: $hornblende-green;
+  border: 1px solid $translucent-silk;
   border-radius: 10px;
   font-size: 1rem;
   font-weight: 300;
   cursor: pointer;
 }
 
 .btn:hover {
-  background-color: $spruce-stone;
+  background-color: $translucent-silk;
+  color: $severely-burnt-toast;
+}
+
+@media only screen and (min-width: 992px) {
```

Comment:
Both of these are valid CSS. But thankfully we are using SCSS which allows nesting. So the better approach would be to colocate all styles for individual components.

So f.e.

```
.btn {
 ... styles for button

  &:hover {
   ... styles for hover button 
}

@media only screen and (min-width: 992px) {
  ... styles for button in this media query
}
}```